.github/website-challenge-md126/Yonis.html

```diff
@@ -0,0 +1,103 @@
+
+<!--First Row-->
+
+        <div class="profile">
+          <h2 id=headingprofile> Profile 
+          </h2>
+          <i class="material-icons">person</i>22 Years Old
+          <br> <br>
+          <i class="material-icons">work</i>Warehouse Operations
+          <br> <br>
+          <i class="material-icons">home</i>Birmingham, United Kingdom
+          </div>
+          <div class="profilepicyonis">
+            <img id="picyonis" src="yonis.jpg" alt="Photo of Yonis">
+        </div>
+        
+
+        <div class="about">
+          <h2 class="h2font"> About
+          </h2>
+          <p> Hi! I was born in Holland but moved to the United Kingdom with my family when I was two&#128747;. I am bilingual and can speak English and Somali fluently and a little Arabic too. 
+          I am beginning my leap into the world of tech through the School of Code, with the end goal to become a front end web developer &#128187;.
+        </p>
+        <img id="tech" src="tech.jpg" alt="Tech photo">
+        </div>
+
+<!--Second Row-->
+
+        <div class="game">
+          <h2 class="h2font" id="gametitle"> Two Truths and One Lie 
+          </h2>
+        <ul id="list">
+          <li>&#128114;&#127998;&#128114;&#127998;I have a twin brother.</li> 
+          <li>&#128214; I have a poem published in a poetry book.</li> 
+          <li class="lie"><span>&#127941;I hold a school record for the fastest ever 100m.</span></li>
+          <li><strong>Can you guess the lie?</strong></li>
+          <li>(Hover the mouse over the answer you think is a lie)
+          </li>
+        </ul>
+      </div>
+
+
+<!--Third Row-->
+
+      <div class="water">
+        <br>
+        <h2 class="h2font">Fitness
+        </h2>
+        <p> &#128170;&#127998;I have a gym membership as I like to keep fit but I usually just end up posing like my picture. I also enjoy being by the water and have been both kayaking and jet skiing.</p>
+        <img id="gym" src="gym.jpg" alt="Photo of Yonis at the gym">
+        <img id="jetski" src="jetski.jpg" alt="Photo of Yonis on a jet ski">
+        <img id="kayak" src="kayaking.jpg" alt="Photo of Yonis kayaking">
+        <br>
+        <br>
+      </div> 
+
+      <div class="dream">
+        <br>
+        <h2 class="h2font">Holiday
+        </h2>
+        <img id="mauritius" src="mauritius.jpg" alt="Photo of mauritius">
+        <p>My dream holiday destination is Mauritius &#128526; but I guess that will have to wait until I find that special someone in my life.</p>
+      </div>
+
+      <div class="scratch">
+        <br>
+        <h2 class="h2font">My Work
+        </h2>
+        <p>Check out my first ever project!</p>
+       <iframe id="scratch-game" allowtransparency="true" width="400" height="300" frameborder="0" allowfullscreen src="https://scratch.mit.edu/projects/499031621/embed"></iframe>
+      <br>
+      <br> 
+      </div> 
+
+ </main>
+
+  <!--Footer-->
+
+  <div class="blocks">
+
+  <div class="footer">Created by
```

Comment:
Semantic code makes for clean, more readable code for developers and machines - but also improves accessibility for users with disabilities. 
```suggestion
  <footer>Created by
```